content/en/docs/tasks/observability/metrics/classify-metrics/index.md

```diff
@@ -35,7 +35,7 @@ For more information, see the
 
 Istio uses the Envoy proxy to generate metrics and provides its configuration in
 the `EnvoyFilter` at
-[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.8.yaml).
```

Comment:
For the version changes, look at https://istio.io/latest/about/contribute/shortcodes/#version-information on how to make this so it will retrieve the current version.